### PR TITLE
2.3.0 Release Notes

### PR DESCRIPTION
**Pull request by jfrontain**
## This PR is a:

- [ x] New topic
- [ ] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will...

## Additional information

List all affected URLs 

- ...
- ...

When this pull request is merged, it will add release notes for the Magento Open Source and Commerce 2.3.0 release.

whatsnew

The  [Magento Commerce 2.3.0 Release Notes](https://devdocs.magento.com/guides/v2.3/release-notes/ReleaseNotes2.3.0Commerce.html) and [Magento Open Source 2.3.0 Release Notes](https://devdocs.magento.com/guides/v2.3/release-notes/ReleaseNotes2.3.0OpenSource.html
)  provide detailed information about the Magento 2.3.0 release.

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=3349) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=3349) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: lorikrell<br/>:x: Jeanne Frontain<hr/>**Jeanne Frontain** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

File: guides/v2.3/release-notes/ReleaseNotes2.3.0Commerce.md [hunk 1]
```diff
@@ -3,626 +3,2178 @@ group: release-notes
...
+
+#### Klarna Payments
+
+* Added descriptive text to the Refund API call 

+
+* Added a link to the Klarna merchant portal 

+
+* Show a detailed Klarna error message in the Magento Admin where needed
```
Comment (nkalinovich) [inline guides/v2.3/release-notes/ReleaseNotes2.3.0Commerce.md]: To be consistent w. previous bulleted points, I believe it's possible to make it: "Added a detailed Klarna message in the Magento admin where needed.

File: guides/v2.3/release-notes/ReleaseNotes2.3.0Commerce.md [hunk 2]
```diff
@@ -3,626 +3,2178 @@ group: release-notes
...
+* Show a detailed Klarna error message in the Magento Admin where needed 

+
+* Added an initial Magento Functional Test Framework (MFTF) test and support for future tests 

+
+* Extended cleanser filtering 

+
+* Added support for PHP 7.2  and dropped support for PHP 5.6
```
Comment (nkalinovich) [inline guides/v2.3/release-notes/ReleaseNotes2.3.0Commerce.md]: Extra space after 7.2

**Comment by jfrontain** [comment]
run tests

**Pull request merged**